src/org/omegat/filters2/moodlephp/MoodlePHPFilter.java

```diff
@@ -135,9 +135,11 @@ protected void processFile(BufferedReader inFile, BufferedWriter outFile, Filter
                 processBlock(block.toString(), outFile);
                 block.setLength(0);
             }
-            if (!Character.isWhitespace(c)) { // In the regexp, there could be whitespace between " and >
-                previousChar = c;
+            if (c == quotes && previousChar == 92) {
+                previousChar = 0;
             }
+            else
```

Comment:
Our coding conventions require brackets for the `else`.

You can run `./gradlew spotlessChangedApply` to run an autoformatter on the files changed in your branch.

Reply:
Done.